Gemfile.lock

```diff
@@ -246,4 +246,4 @@ DEPENDENCIES
   webmock (~> 3.24)
 
 BUNDLED WITH
-   2.3.12
+   2.4.9
```

Comment:
Is there any advantage to using the same bundler version as jupiter? 

https://github.com/ualbertalib/jupiter/blob/caaa54e6932afd8c3c1a92b3a1a26eb6ee4d8b94/Gemfile.lock#L708
```ruby
   2.5.6
```

Reply:
```
There are no versions of bundler (= 2.5.6) compatible with your Ruby & RubyGems
        bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.6.219.
```
I can see what might be required to bump the Ruby version, but I'll leave the bundler version and add a call out to the bundle version change in the Changelog.

Reply:
Sounds good to me.